### PR TITLE
fix: build with Xcode 12.4 SQPIT-781

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-781" title="SQPIT-781" target="_blank">TaskSQPIT-781</a>  Build frameworks compiles on both Xcode 12 and 13
  </td></table>
  <br />
 

# What's new in this PR?

ref: https://github.com/wireapp/wire-ios-protos/pull/60

### CODE REVIEW

File: Tests/Source/ManagedObjectContext/CoreDataStackTests+EncryptionAtRest.swift [hunk 1]
```diff
@@ -26,11 +26,11 @@ class CoreDataStackTests_EncryptionAtRest: DatabaseBaseTest {
...
-#if targetEnvironment(simulator)
-        if #available(iOS 15, *) {
-            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
-        }
-#endif
+//#if targetEnvironment(simulator)
+//        if #available(iOS 15, *) {
+//            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
+//        }
+//#endif
```
Comment (billypchan) [inline Tests/Source/ManagedObjectContext/CoreDataStackTests+EncryptionAtRest.swift:33]:
```suggestion
#if targetEnvironment(simulator)
        if #available(iOS 15, *) && swift(>=5.4) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
//#endif
```

Comment (billypchan) [inline Tests/Source/ManagedObjectContext/CoreDataStackTests+EncryptionAtRest.swift:33]:
```suggestion
#if targetEnvironment(simulator) && swift(>=5.4)
        if #available(iOS 15, *) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
#endif
```

File: Tests/Source/ManagedObjectContext/CoreDataStackTests+EncryptionAtRest.swift [hunk 2]
```diff
@@ -27,10 +27,10 @@ class CoreDataStackTests_EncryptionAtRest: DatabaseBaseTest {
         let sut = createStorageStackAndWaitForCompletion()
         let account = Account(userName: "", userIdentifier: UUID())
 #if targetEnvironment(simulator)
-        if #available(iOS 15, *) {
+        if #available(iOS 15, *) && swift(>=5.4) {
             XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
         }
-#endif
+//#endif
```
Comment (billypchan) [inline Tests/Source/ManagedObjectContext/CoreDataStackTests+EncryptionAtRest.swift:33]:
```suggestion
#endif
```

File: Tests/Source/Utils/EncryptionKeysTests.swift [hunk 3]
```diff
@@ -44,11 +44,11 @@ class EncryptionKeysTests: XCTestCase {
...
-#if targetEnvironment(simulator)
-        if #available(iOS 15, *) {
-            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
-        }
-#endif
+//#if targetEnvironment(simulator)
+//        if #available(iOS 15, *) {
+//            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
+//        }
+//#endif
```
Comment (billypchan) [inline Tests/Source/Utils/EncryptionKeysTests.swift:51]:
```suggestion
#if targetEnvironment(simulator)
         if #available(iOS 15, *) && swift(>=5.4) {
             XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
         }
 #endif
```

File: Tests/Source/Utils/EncryptionKeysTests.swift [hunk 4]
```diff
@@ -45,10 +45,10 @@ class EncryptionKeysTests: XCTestCase {
...
         // given
 #if targetEnvironment(simulator)
-        if #available(iOS 15, *) {
-            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
-        }
-#endif
+         if #available(iOS 15, *) && swift(>=5.4) {
+             XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
+         }
+ #endif
```
Comment (billypchan) [inline Tests/Source/Utils/EncryptionKeysTests.swift:51]:
```suggestion
        if #available(iOS 15, *) && swift(>=5.4) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
#endif
```

File: Tests/Source/Utils/EncryptionKeysTests.swift [hunk 5]
```diff
@@ -61,11 +61,11 @@ class EncryptionKeysTests: XCTestCase {
...
-#if targetEnvironment(simulator)
-        if #available(iOS 15, *) {
-            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
-        }
-#endif
+//#if targetEnvironment(simulator)
+//        if #available(iOS 15, *) {
+//            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
+//        }
+//#endif
```
Comment (billypchan) [inline Tests/Source/Utils/EncryptionKeysTests.swift:68]:
```suggestion
#if targetEnvironment(simulator)
        if #available(iOS 15, *) && swift(>=5.4) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
#endif
```

File: Tests/Source/Utils/EncryptionKeysTests.swift [hunk 6]
```diff
@@ -74,11 +74,11 @@ class EncryptionKeysTests: XCTestCase {
...
```
Comment (billypchan) [inline Tests/Source/Utils/EncryptionKeysTests.swift:81]:
```suggestion
#if targetEnvironment(simulator)
        if #available(iOS 15, *) && swift(>=5.4) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
#endif
```

File: Tests/Source/Utils/EncryptionKeysTests.swift [hunk 7]
```diff
@@ -91,11 +91,11 @@ class EncryptionKeysTests: XCTestCase {
...
```
Comment (billypchan) [inline Tests/Source/Utils/EncryptionKeysTests.swift:98]:
```suggestion
#if targetEnvironment(simulator)
        if #available(iOS 15, *) && swift(>=5.4) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
#endif
```

File: Tests/Source/Utils/EncryptionKeysTests.swift [hunk 8]
```diff
@@ -109,11 +109,11 @@ class EncryptionKeysTests: XCTestCase {
...
```
Comment (billypchan) [inline Tests/Source/Utils/EncryptionKeysTests.swift:116]:
```suggestion
#if targetEnvironment(simulator)
        if #available(iOS 15, *) && swift(>=5.4) {
            XCTExpectFailure("Expect to fail on iOS 15 simulator. ref: https://wearezeta.atlassian.net/browse/SQCORE-1188")
        }
#endif
```

**Review by KaterinaWire — APPROVED** [review]

**Review by johnxnguyen — APPROVED** [review]

File: Cartfile.resolved [hunk 9]
```diff
@@ -6,7 +6,7 @@ github "wireapp/swift-protobuf" "1.15.0_xcframework"
 github "wireapp/wire-ios-cryptobox" "31.0.0"
 github "wireapp/wire-ios-images" "40.0.0"
 github "wireapp/wire-ios-link-preview" "34.0.0"
-github "wireapp/wire-ios-protos" "30.1.0"
+github "wireapp/wire-ios-protos" "fix/build-with-xcode-12.4"
```
Comment (billypchan) [inline Cartfile.resolved:9]:
```suggestion
github "wireapp/wire-ios-protos" "30.1.1"
```

**Comment by billypchan** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release patch** for PR #**1314**. Will create a **patch** release after merged.
Next version is 280.0.6.

**Pull request merged**